packages/block-library/src/table/theme.scss

```diff
@@ -1,5 +1,6 @@
 .wp-block-table {
 	overflow-x: auto;
+	display: block; // Necessary in order for this table to be mobile responsive.
```

Comment:
This seems odd to me, changing `display` of a table element. I had some problem with this rule in a  PR, so I took it out. What Exactly does mobile responsive mean here? Could you elaborate?

Reply:
To answer your question, if we had the whole thing wrapped in a div, we could do this instead:

https://codepen.io/joen/pen/RYojVZ?editors=1100

But as the markup is today, we need to leverage the `table` as the full-width block to hold the horizontal scrollbar, and then make a child element of that (in this case, tbody/thead/tfoot) be the full-wide table that has a min-width.